client/src/styles/FAQ.module.css

```diff
@@ -0,0 +1,42 @@
+.FAQBox {
+    color: var(--theme-title-text);
+    font-size: var(--theme-lg);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: flex-end;
+    padding-top: var(--theme-sm);
+}
+
+.chevron {
+    height: var(--theme-md);
+    color: var(--theme-title-text);
+    transition: 0.5s;
+}
+
+.chevronRotate{
+    transform: rotate(180deg);
+    height: var(--theme-md);
+    transition: 0.5s;
+}
+
+.contentContainer {
+    color: var(--theme-page-text);
+    width: 80%;
+    font-size: 18px;
+    transition: max-height 0.6s ease;
```

Comment:
https://css-tricks.com/using-css-transitions-auto-dimensions/#there-are-two-crucial-downsides-to-this